src/example/DistributedLogical/Distributed_msg.ts

```diff
@@ -0,0 +1,143 @@
+// Code generated by the Lingua Franca compiler from file:
+// /Users/matt.weber/git/lingua-franca/example/DistributedTS/Distributed.lf
```

Comment:
```suggestion
// https://github.com/icyphy/lingua-franca/blob/master/example/Distributed/Distributed.lf
```

Reply:
This could just be removed entirely. These programs started off generated, but I cleaned them up a bit.

Reply:
We should not remove it but instead point to a valid LF source, as I proposed. Without that reference the example suggests people are to write federated reactors by hand, which I certainly wouldn't recommend.